Makefile

```diff
@@ -55,31 +52,40 @@ run: ## Run Karpenter controller binary against your local cluster
 		go run ./cmd/controller/main.go
 
 test: ## Run tests
-	go test -v ./pkg/$(shell echo $(TEST_SUITE) | tr A-Z a-z)/... \
+	go test -v ./pkg/... \
 		-cover -coverprofile=coverage.out -outputdir=. -coverpkg=./... \
 		--ginkgo.focus="${FOCUS}" \
 		--ginkgo.randomize-all \
 		--ginkgo.vv
 
+deflake: ## Run randomized, racing tests until the test fails to catch flakes
+	ginkgo \
+		--race \
+		--focus="${FOCUS}" \
+		--randomize-all \
+		--until-it-fails \
+		-v \
+		./pkg/...
+
 e2etests: ## Run the e2e suite against your local cluster
 	cd test && CLUSTER_ENDPOINT=${CLUSTER_ENDPOINT} \
 		CLUSTER_NAME=${CLUSTER_NAME} \
 		INTERRUPTION_QUEUE=${CLUSTER_NAME} \
 		go test \
 		-p 1 \
 		-count 1 \
-		-timeout ${TEST_TIMEOUT} \
+		-timeout 3h \
```

Comment:
Why go for hardcode values here?

Reply:
I don't think that anyone was ever changing these values, so I didn't see a need for them to be parameterized. If we decide that we want/need them to be parameterized later, we can always add the environment variables back